apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3550,6 +3552,33 @@ static ItemComponent mapEobCommonItemRevenue(
     return item;
   }
 
+  /**
+   * Maps the Revenue Status Indicator Code to the eob's item revenue as an extension, if the status
+   * code is present.
+   *
+   * <p>REV_CNTR_STUS_IND_CD => ExplanationOfBenefit.item.revenue.extension
+   *
+   * @param item the item to add the extension to, if the required data is present
+   * @param eob the root eob (only used for logging purposes)
+   * @param statusCode the status code to check for and add data from if exists
+   * @return the {@link ItemComponent}
+   */
+  static ItemComponent mapEobCommonItemRevenueStatusCode(
+      @Nonnull ItemComponent item, @Nonnull ExplanationOfBenefit eob, Optional<String> statusCode) {
+
+    Assert.notNull(item, "Item must be non-null");
+    Assert.notNull(eob, "Eob must be non-null");
```

Comment:
I like asserting these (an alternative is lombok annotations but it is not widely adopted in BFD outside of PACA -- something that we should discuss when we are working on style guide). My only question is whether there is precedent in other parts of the code for any other null checking assertions that use a different mechanism?

Reply:
From a search I don't see any other specific null checking mechanisms outside of general != null checks in the code logic, so this shouldnt conflict with any existing strategies.